Tools/Scripts/download-github-release.py

```diff
@@ -1,162 +0,0 @@
-#! /usr/bin/env python3
```

Comment:
This looks to be used downstream, so should have that resolved before removing the file.

Reply:
Pretty sure we stopped using GitHub releases internally so this script is of no use to it.